examples/data-sources/datacenters.tf

```diff
@@ -0,0 +1,11 @@
+
+data "fastly_datacenters" "fastly" {}
+
+output "fastly_datacenters_all" {
+  value = data.fastly_datacenters.fastly
```

Comment:
Should this be... ?

```suggestion
  value = data.fastly_datacenters.fastly.pops
```

Reply:
in order to represent the whole structure, I didn't include `.pop` and I think that's right.